tests/cypress/plugins/unpackZipArchive/addPlugin.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+// eslint-disable-next-line no-use-before-define
+exports.unpackZipArchive = unpackZipArchive;
+
+const path = require('path');
+const extract = require('extract-zip');
+
+// eslint-disable-next-line consistent-return
+async function unpackZipArchive(args) {
+    const { arhivePath } = args;
+    const absolutePath = path.dirname(path.resolve(arhivePath));
+    try {
+        await extract(arhivePath, { dir: absolutePath });
+        return null;
```

Comment:
Probably should be placed out of try block to fix the eslint issue ``consistent-return``

Reply:
Fixed.